src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,14 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import { ArrayOfTodos } from '../types/Todos';
+
+type Props = {
+  todos: ArrayOfTodos
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <>
+    {todos.map(todo => (
+      <TodoInfo todo={todo} key={todo.id} />
+    ))}
+  </>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings